chromium_src/components/os_crypt/os_crypt_win.cc

```diff
@@ -0,0 +1,17 @@
+#include "base/command_line.h"
+#include "brave/common/brave_switches.h"
+
+// switches::kDisableEncryption
+const char kDisableEncryption[] = "disable-encryption";
+
+namespace {
+bool IsEncryptionDisabled(const std::string input_text, std::string* output_text) {

```

Comment:
sorry, I screwed up and it should be `const std::string& input_text` for the most efficient argument passing.

Reply:
Done :)